src/store/slices/ui-slice.ts

```diff
@@ -3,11 +3,12 @@ import { createSlice } from '@reduxjs/toolkit';
 const UiSlice = createSlice({
   name: 'ui',
   initialState: {
-    theme: 'dark',
+    theme: 'light',
```

Comment:
```suggestion
    theme: 'light',
```
switch default theme back to light